index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link
+      rel="stylesheet"
+      media="screen"
+      href="https://fontlibrary.org//face/poppins"
+      type="text/css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <style>
+      body,
+      p {
+        font-family: "Roboto", sans-serif;
+      }
+    </style>
+    <link href="http://fonts.cdnfonts.com/css/crete-round" rel="stylesheet" />
+
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <section class="header">
+      <div>
+        <p class="logo">John Doe</p>
+      </div>
+      <div class="menu_btn">
+        <div class="menu"></div>
+        <div class="menu"></div>
+        <div class="menu"></div>
+      </div>
```

Comment:
- [ ] Make sure that you use [Semantic HTML](https://www.w3docs.com/learn-html/semantic-elements-in-html5.html). Kindly,  add the hamburger button inside a `<nav>` tag